src/views/EditNote/actions/EditNote.actions.ts

```diff
@@ -85,7 +85,7 @@ export const createNoteRequest = note => dispatch => {
         data: note
     })
     .then(response => dispatch(handleSuccessfulNoteCreation(response.data)))
-    .catch(error => dispatch(handleFailedNoteCreation(error)));
+    .catch(error => dispatch(handleFailedNoteCreation('sorry, e')));
```

Comment:
Hardcoded stuff? You can just dispatch an action with no payload (only type) and hardcode the value in reducer.